lib/Models/DateTimeParameter.js

```diff
@@ -44,4 +44,22 @@ defineProperties(DateTimeParameter.prototype, {
      */
 });
 
+/**
+ * Process value so that it can be used in an URL.
+ * @param {String} value Value to use to format.
+ * @return {String} Stringified JSON that can be used to pass parameter value in URL.
+ */
+DateTimeParameter.formatValueForUrl = function(value) {
+    return JSON.stringify({
+                type: 'object',
```

Comment:
This is... weird.  I mean, what is this format?  It seems like it might be WPS specific, in which case it doesn't belong in the DateTimeParameter.  I realize this ship has already sailed a bit, but the whole notion of `formatValueForUrl` doesn't make a lot of sense to me.  For the GeoJSON types, it seems like it does the same thing as `getGeoJsonFeature` but then wraps it up in a FeatureCollection and JSON.stringify's it.  It would be much better IMO to be specific about this.  Have a function `getGeoJsonFeatureCollection`, and if someone wants that as a JSON string they can call JSON.stringify themselves.